### ISSUE TITLE
Bug for 3d

### ISSUE THREAD

**Issue by ycwu1997** [VainF/Pytorch-Synchronized-BatchNorm#2]
Good implementation, but note that to change the dimension for 3D tensors.

        B, C,H,W,D = grad_output.shape
&     grad_x.view(B, C,H,W,D)

**Comment by VainF** [comment]
Hi ycwu1997, Thank you for pointing out the problem. The wrong shape was fixed in the latest commit.
